class.jetpack.php

```diff
@@ -603,14 +603,22 @@ function jetpack_jitm_ajax_callback() {
 		if ( isset( $_REQUEST['jitmActionToTake'] ) && 'dismiss' == $_REQUEST['jitmActionToTake'] ) {
 			// get the hide_jitm options array
 			$jetpack_hide_jitm = Jetpack_Options::get_option( 'hide_jitm' );
-			$module_slug = $_REQUEST['jitmModule'];
+			$jitm_name = $_REQUEST['jitmName'];
 
-			if( ! $jetpack_hide_jitm ) {
+			if( ! $jetpack_hide_jitm || ! is_array( $jetpack_hide_jitm ) ) {
 				$jetpack_hide_jitm = array(
-					$module_slug => 'hide'
+					$jitm_name => array (
+						'last_dismissed' => time(),
+						'dismiss_count'  => 1
+				) );
```

Comment:
These closing parenths should be on separate lines, first one indented one more tab.